pages/15.contributing/11.Contributing-financially/docs.md

```diff
@@ -12,4 +12,8 @@ taxonomy:
 ---
 
 ---
-## Contributing financially to Mautic
\ No newline at end of file
+## Contributing financially to Mautic
+
+If you desire to contribute and support the project and have a say in the future of Mautic please see https://www.mautic.org/become-a-member-of-mautic. There are individual and corporate packages available. 
```

Comment:
```suggestion
If you desire to contribute and support the project and have a say in the future of Mautic please see [Become a member of Mautic[member]. There are individual and corporate packages available. 
```

This is how we should do link macros with Grav - we're in the process of moving this over to Read the Docs so it will eventually be done much more elegantly! I've added the macros in the second comment.